src/modules/app/hooks/useActionForToken.tsx

```diff
@@ -19,92 +21,123 @@ export const useActionForToken = () => {
   const actionForToken = useCallback(
     (symbol: string, balance: string) => {
       const { LinkedAction, InputAmount, SourceToken, TargetToken, Widget, Locale, Details } = QueryParams;
-      const { REWARDS_INTENT: REWARD, UPGRADE_INTENT: UPGRADE, TRADE_INTENT: TRADE } = IntentMapping;
+      const {
+        REWARDS_INTENT: REWARD,
+        UPGRADE_INTENT: UPGRADE,
+        TRADE_INTENT: TRADE,
+        SAVINGS_INTENT: SAVINGS
+      } = IntentMapping;
       const retainedParams = [Locale, Details];
 
       const lowerSymbol = symbol.toLowerCase();
+      const upperSymbol = symbol.toUpperCase();
+      const image = `/tokens/actions/${lowerSymbol}.png`;
```

Comment:
You can use `useTokenImage` and it will return the best available option, prioritizing `svg`.

Reply:
> You can use `useTokenImage` and it will return the best available option, prioritizing `svg`.

I don't think we're exporting `useTokenImage` from helipad

Reply:
Going to merge this, we can make this update separate PR if we think it's a good idea